src/systemd/sd-journal.h

```diff
@@ -87,7 +87,7 @@ int sd_journal_open_namespace(sd_journal **ret, const char *name_space, int flag
 int sd_journal_open_directory(sd_journal **ret, const char *path, int flags);
 int sd_journal_open_directory_fd(sd_journal **ret, int fd, int flags);
 int sd_journal_open_files(sd_journal **ret, const char **paths, int flags);
-int sd_journal_open_files_fd(sd_journal **ret, int fds[], unsigned n_fds, int flags);
+int sd_journal_open_files_fd(sd_journal **ret, const int fds[], unsigned n_fds, int flags);
```

Comment:
We discussed constification before, and it was rejected because of the C++ symbol breakage. Did something change? Maybe this should only be done for non-C++ using some macro magic?

Reply:
hmm, but this is `extern "C"` code anyway, no? And adding the "const" just means weaker requirements on the variable passed. I really thought this was safe to change... But I could also just drop it or change into a comment...

Reply:
I asked the interwebs for help on this one: https://twitter.com/pid_eins/status/1223206955021410305

Reply:
So, yeah, just learnt from that: it is an API break, though not an ABI one, since taking a ptr of the function and assigning it to some variable would result in incompatible pointers before and after the change. Hence, let's drop this.